File: Problem Statements/Temperature Conversion/F - K/F2K.c
#include<stdio.h>
void main(){
    double k,f=0.0;
    printf("Enter Temperature (in Fahrenheit) : ");
    scanf("%lf",&f);
    k=((f-32)*5/9)+273.15;
    printf("Temperature in Kelvin : %0.2f",k);
}

File: Problem Statements/Temperature Conversion/F - K/F2K.java

import java.util.Scanner;
class F2K{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter Temperature (in Fahrenheit) : ");
        double f=sc.nextDouble();
        sc.close();
        double k=((f-32)*5/9)+273.15;
        System.out.print("Temperature in Kelvin : "+k);
    }
}